caom2pipe/manage_composable.py

```diff
@@ -1861,7 +1874,9 @@ def set_product_id(self, **kwargs):
     def remove_extensions(name):
         """How to get the file_id from a file_name."""
         return (
-            name.replace('.fits', '').replace('.gz', '').replace('.header', '')
+            name.replace('.fits', '')
```

Comment:
If this is just a root of the file name it could be replaced with `name.split('.')[0]`.

Reply:
CFHT, OMM, and VLASS all have a possibility of file names with dots in them, which is why the implementation is multiple `replace` calls. Maybe a regex would be faster?

Reply:
Forgot about dots in the name. Please ignore my comment.